fastlane/lanes/localization.rb

```diff
@@ -178,10 +203,6 @@ def ensure_default_metadata_are_not_overridden(metadata_files_hash:)
     MSG
     UI.user_error!(override_not_allowed_message) if File.exist?(en_file_path)
   end
-
-  lane :lint_localizations do
-    ios_lint_localizations(input_dir: APP_RESOURCES_DIR, allow_retry: true)
-  end
```

Comment:
Don't know how this ended up inside a method 😳

Reply:
Ooof. Nice catch.

Probably bad indentation at some point in the past tricked us, and then we ran `rubocop -A` at some point later but didn't notice it despite being more obvious once indentation was fixed? 🤷